packages/atoms/src/Hint.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Text as RNText, View } from 'react-native';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+import { Icon } from './Icon/Icon';
+import { IconType } from './Icon/iconTypes';
+import { Color, colorVariants } from '@trezor/theme';
+import * as CSS from 'csstype';
+
+type HintType = 'hint' | 'error';
+
+type HintProps = {
+    type: HintType;
```

Comment:
I know I proposed `type` but as I see it now, maybe `variant` will be better name for this prop